src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length-31 calendar--start-day--tue">
+      <div class="calendar_day calendar_day--day-1"></div>
+      <div class="calendar_day calendar_day--day-2"></div>
+      <div class="calendar_day calendar_day--day-3"></div>
+      <div class="calendar_day calendar_day--day-4"></div>
+      <div class="calendar_day calendar_day--day-5"></div>
+      <div class="calendar_day calendar_day--day-6"></div>
+      <div class="calendar_day calendar_day--day-7"></div>
+      <div class="calendar_day calendar_day--day-8"></div>
+      <div class="calendar_day calendar_day--day-9"></div>
+      <div class="calendar_day calendar_day--day-10"></div>
+      <div class="calendar_day calendar_day--day-11"></div>
+      <div class="calendar_day calendar_day--day-12"></div>
+      <div class="calendar_day calendar_day--day-13"></div>
+      <div class="calendar_day calendar_day--day-14"></div>
+      <div class="calendar_day calendar_day--day-15"></div>
+      <div class="calendar_day calendar_day--day-16"></div>
+      <div class="calendar_day calendar_day--day-17"></div>
+      <div class="calendar_day calendar_day--day-18"></div>
+      <div class="calendar_day calendar_day--day-19"></div>
+      <div class="calendar_day calendar_day--day-20"></div>
+      <div class="calendar_day calendar_day--day-21"></div>
+      <div class="calendar_day calendar_day--day-22"></div>
+      <div class="calendar_day calendar_day--day-23"></div>
+      <div class="calendar_day calendar_day--day-24"></div>
+      <div class="calendar_day calendar_day--day-25"></div>
+      <div class="calendar_day calendar_day--day-26"></div>
+      <div class="calendar_day calendar_day--day-27"></div>
+      <div class="calendar_day calendar_day--day-28"></div>
+      <div class="calendar_day calendar_day--day-29"></div>
+      <div class="calendar_day calendar_day--day-30"></div>
+      <div class="calendar_day calendar_day--day-31"></div>
```

Comment:
```suggestion
      <div class="calendar__day calendar__day--day-31"></div>
```
follow BEM, use double underline